### PR TITLE
Fix the broken blog feed

### LINKED ISSUE #1777: Blog feed is a mix of RSS and Atom
The `feed.xml` file in the gh-pages branch appears to be some weird mixture of RSS and Atom. For example, one of the very first XML elements states that the feed is both RSS version 2.0 and Atom, which is weird.

    <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">

This tag is an Atom link that references self as being an RSS feed, which is odd.

    <atom:link href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}" rel="self" type="application/rss+xml" />

`pubDate` is an RSS element that should contain a date in RFC 822 format, which is correct. Atom XML files do not provide a `pubDate` element, and dates are supposed to be provided in RFC 3339 format.

    <pubDate>{{ post.date | date_to_rfc822 }}</pubDate>

And so on. I suggest migrating to the Atom format. Atom is supposed to be the better spec. It should also have great client support, as the spec was published in 2007.

### PR DESCRIPTION
**Pull request by Ichimonji10**
The links in the blog feed do not correctly link to blog posts, and the feed
itself is a weird mish-mash of RSS and Atom. Do the following:

* Rewrite `feed.xml` in Atom format. Use correct tags and format all dates as
  per RFC 3339.
* Add `author` tags to the YAML front matter at the beginning of each blog post.
* Tweak the site URL in `_config.yml`.

The resultant blog feed validates against http://validator.w3.org/feed/.

Fix #1777.

### CODE REVIEW

File: _posts/2014-12-12-better-test-result-output.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ layout: post
 title: "Better test result output"
 date: 2014-12-12 14:50:00
 categories: update
+author: Elyézer Rezende
```
Comment (elyezer) [inline _posts/2014-12-12-better-test-result-output.md]: Thanks for the `é` :)

**Comment by elyezer** [comment]
Very important change, merging right now.

**Comment by elyezer** [comment]
Karma given

File: _posts/2014-12-12-better-test-result-output.md (hunk 1 above)
Comment (Ichimonji10) [inline _posts/2014-12-12-better-test-result-output.md]: You're welcome. :smile:

**Comment by Ichimonji10** [comment]
The updated blog feed functions correctly. You can see the feed [here](https://satelliteqe.github.io/robottelo/feed.xml).

**Comment by Ichimonji10** [comment]
Follow-up pull request: #1811.

**Pull request merged**